### PR TITLE
CB-7464 - Env is startable/stoppable but it does not exists on provider side

### PR DESCRIPTION
**Pull request by gregito**
CB-7464 - Env is startable/stoppable but it does not exists on provider side

### CODE REVIEW

**Comment by biharitomi** [comment]
@gregito  Looks ok, but there is no test just test failure :)

**Comment by gregito** [comment]
> Looks ok, but could you please cover the new service logics with unit tests.

sure thing!

**Comment by gregito** [comment]
> Looks ok, but could you please cover the new service logics with unit tests.

tests done!

File: environment/src/test/java/com/sequenceiq/environment/environment/flow/start/handler/StartFreeIpaHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        when(mockEnvironmentDtoEvent.getData()).thenReturn(mockEnvironmentDto);
+        when(mockEnvironmentDto.getResourceCrn()).thenReturn(MOCK_ENV_CRN);
+        when(mockEnvironmentDtoEvent.getHeaders()).thenReturn(mockEventHeaders);
+
+        underTest = new StartFreeIpaHandler(mockEventSender, mockFreeIpaPollerService, mockFreeIpaService);
+    }
+
+    @Test
+    void testWhenFreeIpaDescribeTellsFreeIpaIsNotStartableThenExceptionComesAndNoStartEventHappens() {
```
Comment (biharitomi) [inline environment/src/test/java/com/sequenceiq/environment/environment/flow/start/handler/StartFreeIpaHandlerTest.java:65]: Maybe a test for the happy flow to be sure that is still working :)

Comment (gregito) [reply environment/src/test/java/com/sequenceiq/environment/environment/flow/start/handler/StartFreeIpaHandlerTest.java:68]: done but waits for this pr: https://github.com/hortonworks/cloudbreak/pull/8255

**Pull request merged**